src/axom/core/docs/sphinx/core_acceleration.rst

```diff
@@ -7,25 +7,38 @@
 Core acceleration
 ******************************************************
 
-Axom's `core` component provides several utilities for user applications that intend
-to support execution on hardware accelerators.
-Axom lets users control execution space using RAJA and memory space using Umpire.
-As noted in the `RAJA documentation <https://raja.readthedocs.io/en/main/index.html>`_,
-developers of high-performance computing applications have many options for running
-code: on the CPU, using OpenMP, or on GPU hardware accelerators, and the options are
-constantly developing.  RAJA controls *where* code runs; Umpire moves data between
-memory spaces.
+Axom's `core` component provides several utilities for user applications that 
+support execution on hardware accelerators. Axom lets users control execution 
+space using RAJA and memory space using Umpire. As noted in the 
+`RAJA documentation <https://raja.readthedocs.io/en/main/index.html>`_,
+developers of high-performance computing applications have many options for 
+running code: on the CPU, using OpenMP, or on GPU hardware accelerators, and 
+the options are constantly evolving. RAJA controls *where* code runs; Umpire 
+moves data between memory spaces.
```

Comment:
I like this paragraph, but the order of these sentences seems a bit awkward to me for readers who aren't familiar with "execution space" and "memory space".
What do you think about the following (which reworks the final sentence into the second one):
```suggestion
Axom's `core` component provides several utilities for user applications that 
support execution on hardware accelerators. Axom lets users control the execution 
space (i.e. *where* code runs) using RAJA and movement between memory spaces using Umpire. 
As noted in the `RAJA documentation <https://raja.readthedocs.io/en/main/index.html>`_,
developers of high-performance computing applications have many options for 
running code: on the CPU, using OpenMP, or on GPU hardware accelerators, and 
the options are constantly evolving.
```